contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -430,6 +430,10 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
     // to give the callee their requested amount.
     bool success = callWithExactGas(rc.callbackGasLimit, rc.sender, resp);
 
+    // Increment the req count for fee tier selection.
```

Comment:
I don't think we have fee tiers in V2+ (unless we add them back). Can remove this comment in this case.

Reply:
Updated the comment